docs/i18n-languages/spanish/CONTRIBUTING.md

```diff
@@ -23,14 +23,14 @@ Puedes optar por contribuir a cualquier área de tu interés:
 
 2. Ayuda a los campistas en nuestro [foro público](https://www.freecodecamp.org/forum/):
 
-    - ❓ [Aportando una retroalimentación](https://www.freecodecamp.org/forum/?max_posts=1).
-    - 💬 [Aportando comentarios](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1) a sus proyectos.
+    - ❓ [Respondiendo sus cuestiones](https://www.freecodecamp.org/forum/?max_posts=1).
```

Comment:
Respondiendo a sus preguntas
(missing 'a', 'cuestion' is a false friend from 'question')

Reply:
> Respondiendo a sus preguntas
> (missing 'a', 'cuestion' is a false friend from 'question')

I don't understand this one, "cuestiones" is also use as "preguntas". I'm on it with the missing 'a'

Reply:
`Cuestion` is usually translated from subject or matter. Or when in the 'asking' context, it's something you ask to learn the truth about that something. If you're asking with the purpose of learning about a topic, 'pregunta' is the correct term.
https://dle.rae.es/?id=TzZfwOK
https://dle.rae.es/?id=Bb1zrQ1